packages/ai/src/core.ts

```diff
@@ -68,11 +63,17 @@ export class EdgeDBAI {
     });
   }
 
-  private async fetchRag(request: Omit<RAGRequest, "model" | "prompt">) {
+  private async fetchRag(request: any) {
```

Comment:
Is there something more useful than `any` we can use here? Like maybe `unknown`?

Reply:
tnx, I updated this